feedly/api_client/data.py

```diff
@@ -76,6 +76,24 @@ def tag_entries(self, entry_ids: List[str]):
         self._client.do_api_request(f'/v3/tags/{quote_plus(self["id"])}', method='put',
                                     data={'entryIds': [entry_id for entry_id in entry_ids]})
 
+    def delete_tags(self, options: StreamOptions = None):
+        """
+        *** WARNING *** Non-reversible operation
+        Given a TagBas Streamable, remove tags corresponding to this tag stream, whichever user saved it, for
```

Comment:
"whichever user saved it" may be unclear, especially since most non-feedly devs won't be too familiar with the teams product.

Reply:
Ok I update the description then